### PR TITLE
Make theme able to load custom javascript

### PR DESCRIPTION
**Pull request by david-alex-cohen-zone**
- Look for the field 'customScript' in editorTheme (settings.js)
- Add it to mustach context
- Load list on template side (index.mst)

### CODE REVIEW

File: red/api/theme.js [hunk 1]
```diff
@@ -128,6 +128,15 @@ module.exports = {
             }
         }
 
+        if (theme.customScripts) {
```
Comment (knolleary) [inline red/api/theme.js]: The `editorTheme` property already has the ability to inject CSS in the page via `editorTheme.page.css`. This change should be consistent with that and use the property name `editorTheme.page.scripts`.

File: red/api/theme.js [hunk 2]
```diff
@@ -128,6 +128,15 @@ module.exports = {
...
+            var urls = []
+            for (var j = 0; j < theme.customScripts.length; j++) {
```
Comment (knolleary) [inline red/api/theme.js]: The user should be allowed to specify either a single string or an array of strings.

**Comment by david-alex-cohen-zone** [comment]
Thanks for review :) I made some fixes

**Comment by knolleary** [comment]
Retargeted for the 0.17 branch as it's new function rather than issue fixing.

Thanks!

**Pull request merged**